lib/generators/stimulus_reflex/templates/app/javascript/config/mrujs.js.tt

```diff
@@ -0,0 +1,9 @@
+import CableReady from "cable_ready"
+import mrujs from "mrujs"
+import { CableCar } from "mrujs/plugins"
+
+mrujs.start({
+  plugins: [
+    new CableCar(CableReady)
+  ]
+})
```

Comment:
I don't think we should include mrujs in the StimulusReflex installer, since that's not part of SR's scope.

Reply:
I proposed this with the intention of starting a discussion! 💯

I speculate that you're, but it's an important point that the `mrujs` step is presented as optional, with an explanation offered. I definitely do not plan to try and sneak it in the back door, but offering folks an opinionated nudge is IMO a very Rails-y thing to do. To put the ball back in your court for a moment... would you feel differently if the prompt to swap was only presented to people who currently have `@rails/ujs` in their deps?

So, here's my pitch: as much as I'm mildly obsessed with Action Cable, there are still things that WS cannot do. `mrujs` has been the tool that has allowed me to do them anyhow. The most significant example is Devise session creation and destruction without the need for a redirect or hard refresh. Another example is setting secure cookies. I propose that `mrujs` can be used as a polyfill to give WS-powered apps the ability to leverage Action Dispatch resources an API that people have been using for a decade.

The icing on top is, of course, the CableCar plugin, which I believe is an excellent marketing flywheel. Keep in mind that the SR installer is also (in my vision, at least) a prototype for a stripped down version that will be a CR installer. I would argue that the connection to SR is still unproven, the opportunity for CR/CC is all upside. Clicking a button and receiving a CR broadcast is still amazing. I honestly believe that if CR/mrujs/CC had been available in 2019, there would have been no need for Turbo. Oh, well?

Anyhow, this is ultimately not my decision to make. People have been bundling software for a long time, and historically it's been an incredibly effective approach. Think about games and GPUs, or modems and fax software. The hanger-on gets distribution and credibility, and the host gets to show that it has an ecosystem.

Bluntly, being able to demonstrate that we have an ecosystem might be one of the most potent signals we can send to the people who refuse to understand why we bother to exist when Turbo is already in their new Rails project. As someone said on Discord, people make technology decisions largely based on their perception that other people are using it, too. We need this! So why not lift the tide for all boats?